Packs/Slack/pack_metadata.json

```diff
@@ -3,6 +3,7 @@
     "description": "Interact with Slack API - collect logs, send messages and notifications to your Slack team.",
     "support": "xsoar",
     "currentVersion": "3.1.44",
+    "currentVersion": "3.1.43",
```

Comment:
@maimorag Also looks like some funky behavior from the update-release-notes command.